bc/subscription/tasks.py

```diff
@@ -8,15 +13,34 @@ def process_filing_webhook_event(fwe_pk) -> FilingWebhookEvent:
     :return: A FilingWebhookEvent object that was updated.
     """
     filing_webhook_event = FilingWebhookEvent.objects.get(pk=fwe_pk)
```

Comment:
This all looks good. One last thing is that you'll want it wrapped in a transaction.